src/templates/gwfcore/gwfcore-launch-template.template.yaml

```diff
@@ -94,15 +94,72 @@ Resources:
                 --==BOUNDARY==
                 Content-Type: text/cloud-config; charset="us-ascii"
 
+                #cloud-config
+                repo_update: true
+                repo_upgrade: all
```

Comment:
`repo_update` and `repo_upgrade` were required to install `amazon-cloudwatch-agent`; see https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/user-data.html